velox/type/Type.cpp

```diff
@@ -233,6 +235,14 @@ TypePtr Type::create(const folly::dynamic& obj) {
   }
 }
 
+// static
+void Type::registerSerDe() {
+  velox::DeserializationRegistryForSharedPtr().Register(
```

Comment:
nit: why aren't these methods names lower camel case?

Reply:
+1

Reply:
This is legacy from Koski. We may want to clean this up at some point.